system/jhtree/ctfile.hpp

```diff
@@ -249,8 +249,9 @@ class jhtree_decl CJHTreeNode : public CNodeBase
 // reading methods
     offset_t prevNodeFpos() const;
     offset_t nextNodeFpos() const ;
-    virtual bool getKeyAt(unsigned int num, char *key) const;         // Retrieve keyed fields
-    virtual bool getValueAt(unsigned int num, char *key) const;       // Retrieve full record
+    inline bool isKeyAt(unsigned int num) { return (num < hdr.numKeys); }
```

Comment:
I think calls to isValueAt could call this instead and avoid a virtual call.

Reply:
There are no calls to isValueAt... I will remove it